app/marketing/apps.py

```diff
@@ -24,3 +24,6 @@
 class MarketingConfig(AppConfig):
 
     name = 'marketing'
+
+    def ready(self):
+        import marketing.signals  # noqa
```

Comment:
Can we be explicit here and define: `from .signals import create_email_subscriber` and remove the `noqa`